docs/user-guide/examples/supported_language_examples_index.md

```diff
@@ -0,0 +1,12 @@
+# Examples in Supported Languages
+
+The User Guide Examples are primarily written in Python due to its popularity but HELICS provides language bindings for many other languages. To demonstrate this, [one of the examples](./advanced_examples/advanced_default) has been written in each of the supporting languages to show how the APIs look in each of those languages but achieve similar functionality.
+
+- [Python (C-API)](https://github.com/GMLC-TDC/HELICS-Examples/tree/main/user_guide_examples/advanced/advanced_default)
+- [Python (Pythonic API) - forthcoming](https://github.com/GMLC-TDC/HELICS-Examples/issues/76)
+- [MATLAB](https://github.com/GMLC-TDC/HELICS-Examples/tree/main/user_guide_examples/advanced/advanced_default_matlab)
```

Comment:
This link doesn't work.

Reply:
Resolved by https://github.com/GMLC-TDC/HELICS-Examples/pull/73

Reply:
I just merged the PR in HELICS-Examples that contains the example; it should work now.